docs/sources/operators-guide/configure/setting-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,138 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/setting-helm-ha-deduplication-consul/
+description: Learn how to configure Grafana Mimir Helm Chart to handle HA Prometheus server deduplication with Consul.
+menuTitle: Configuring high-availability deduplication with Consul
+title: Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+weight: 70
+---
+
+# Configuring mimir-distributed Helm Chart for high-availability deduplication with Consul
+
+Grafana Mimir can deduplicate data from high-availability (HA) Prometheus setup. In this guide, you will see how to configure
+the deduplication for Grafana Mimir helm deployment using external Consul. You also can review 
+[Configuring High Availability]({{< relref "../configuring-high-availability-deduplication.md" >}}) document for some 
+conceptual background.
+
+## Before you begin
+
+You need to have a Grafana Mimir installed.
+
+Refer to [Configuring High Availability]({{< relref "../configuring-high-availability-deduplication.md" >}}) documents
+for high level description on the concept how Mimir deduplicate incoming HA samples.
```

Comment:
```suggestion
For conceptual information about how Mimir deduplicates incoming HA samples, refer to [Configure high availability]({{< relref "../configure-high-availability-deduplication.md" >}}).
```
The information types need to be made more obvious. I will take another pass at this to refactor the documentation.